### PR TITLE
DOCS-2827: Add resource card trigger menu instructions

### PR DESCRIPTION
**Pull request by sguequierre**
* adds instructions to add triggers from resource card context menu

### CODE REVIEW

File: docs/configure/triggers.md [hunk 1]
```diff
@@ -62,6 +62,18 @@ Edit the **Time between notifications** attribute to set the interval at which t
 
    ![The trigger configured with an example URL in the Viam app.](/build/configure/trigger-configured.png)
 
+{{% /tab %}}
+{{% tab name="Builder mode: Resource card" %}}
+You can also configure a trigger for a resource in **Builder** mode on the **CONFIGURE** tab by navigating to the resource's configuration panel, selecting the **...** menu in the upper right corner, and selecting **Create trigger**.
+
+{{<imgproc src="/build/configure/resource-card-create.png" resize="x400" declaredimensions=true alt="The ... menu with Create trigger in the middle of the list of options." >}}
+
+This creates a trigger that's configured to activate on an event occurring from a specific resource.
```
Comment (npentrel) [inline docs/configure/triggers.md:71]:
```suggestion
This creates a trigger that's configured to fire on an event occurring from a specific resource.
```

File: docs/configure/triggers.md [hunk 2]
```diff
@@ -62,6 +62,18 @@ Edit the **Time between notifications** attribute to set the interval at which t
...
+You must configure the method of activation and any relevant **Conditions**, as well as any **Webhooks** and **Emails** for notifications.
+For example, the following triggers when data is ingested from the camera's `GetImages` method:
```
Comment (npentrel) [inline docs/configure/triggers.md:73]:
```suggestion

For example, the following triggers when data is ingested from the camera's `GetImages` method:
```

**Review by npentrel — APPROVED** [review]
Since you've already done it let's merge it but this probably falls under the "UIs people should be able to just use without docs" bucket where we could've left this undocumented.

**Comment by JessamyT** [comment]
I took the liberty of fixing the ticket number since it was linked to a different ticket that I'm working on :D

File: assets/build/configure/resource-card-create.png
Comment (npentrel) [inline assets/build/configure/resource-card-create.png:1]: I think the screenshots need to change to show a sensor rather than a camera

Comment (sguequierre) [reply assets/build/configure/resource-card-create.png:1]: updated

**Comment by npentrel** [comment]
Great, then I think this can merge right?

**Pull request merged**